smart-contracts/contracts/mixins/MixinDisableAndDestroy.sol

```diff
@@ -8,15 +8,15 @@ import 'openzeppelin-eth/contracts/ownership/Ownable.sol';
  * @title Mixin allowing the Lock owner to disable a Lock (preventing new purchases)
  * and then destroy it.
  * @author HardlyDifficult
- * @dev `Mixins` are a design pattern seen in the 0x contracts.  It simply 
- * separates logically groupings of code to ease readability. 
+ * @dev `Mixins` are a design pattern seen in the 0x contracts.  It simply
+ * separates logically groupings of code to ease readability.
  */
 contract MixinDisableAndDestroy is
   IERC721,
   Ownable
 {
   // Used to disable payable functions when deprecating an old lock
-  bool public isAlive; 
```

Comment:
Hmm does our linters not run correctly on Travis?

Reply:
not sure...